packages/front-end/src/app/components/EGRunPipelineFormReviewPipeline.vue

```diff
@@ -53,9 +53,24 @@
           paramsText: paramsText,
         },
       };
-      await $api.seqeraRuns.createPipelineRun(labId, launchRequest);
-      delete runStore.wipSeqeraRuns[seqeraRunTempId];
-      emit('has-launched');
+
+      const res = await $api.seqeraRuns.createPipelineRun(labId, launchRequest);
+
+      if (res) {
```

Comment:
I think it would be better to invert this to the usual pattern and exit early on error:

```
if (!res) {
  throw new Error(...)
}
```